src/utilities/theme/types.ts

```diff
@@ -17,12 +17,24 @@ export interface ThemeColor {
   [key: string]: string;
 }
 
+export type CSSProperties = {
+  [key: string]: string;
+};
+
 export interface TopBar extends ThemeColor {
   background: string;
 }
 
 export type ThemeColors = {
-  topBar: TopBar;
+  topBar?: TopBar;
+  surface?: string;
+  onSurface?: string;
+  interactive?: string;
+  branded?: string;
+  critical?: string;
+  warning?: string;
+  highlight?: string;
+  success?: string;
```

Comment:
It's unlikely we'll expose all these roles on the `ThemeProvider`, but these are all the new color roles added as part of the Color System project.

Reply:
If these might change at all, we'll want to ship this as part of a `v5.0.0` alpha release, in case anyone happens upon them and starts using them for some reason.

Reply:
Updated the interface name here to be `UNSTABLE_colors` so we can make otherwise breaking changes at will.